cmd/invoke.go

```diff
@@ -156,12 +157,24 @@ func runInvoke(cmd *cobra.Command, args []string, newClient ClientFactory) (err
 	}
 
 	// Invoke
-	s, err := client.Invoke(cmd.Context(), cfg.Path, cfg.Target, m)
+	metadata, body, err := client.Invoke(cmd.Context(), cfg.Path, cfg.Target, m)
 	if err != nil {
 		return err
 	}
 
-	fmt.Fprintln(cmd.OutOrStderr(), s)
+	// Print metadata (headers for HTTP, event fields for CloudEvents) if in
+	// verbose mode.
+	if cfg.Verbose {
+		for k, vv := range metadata {
+			values := strings.Join(vv, ";")
+			fmt.Fprintf(cmd.OutOrStdout(), "\"%v\":%v\n", k, values)
```

Comment:
It's just a style thing, but I don't think surrounding the header name with quotes is necessary. To me, it muddies the display. And I think there should be a space after the separating `:`. Compare:

```
❯ func invoke -v
"Access-Control-Allow-Origin":*
"Content-Length":132
"Date":Tue, 05 Apr 2022 20:09:10 GMT
"Connection":keep-alive
"Keep-Alive":timeout=5
"Content-Type":application/json; charset=utf8
"Access-Control-Allow-Methods":OPTIONS, GET, DELETE, POST, PUT, HEAD, PATCH

{"ContentType":"text/plain","Data":"Hello World","ID":"42fd8604-75db-46d3-bdc1-39b3c213595e","Source":"/boson/fn","Type":"boson.fn"}
```
With
```
❯ func invoke -v
Access-Control-Allow-Origin: *
Content-Length: 132
Date: Tue, 05 Apr 2022 20:09:10 GMT
Connection: keep-alive
Keep-Alive: timeout=5
Content-Type: application/json; charset=utf8
Access-Control-Allow-Methods: OPTIONS, GET, DELETE, POST, PUT, HEAD, PATCH

{"ContentType":"text/plain","Data":"Hello World","ID":"42fd8604-75db-46d3-bdc1-39b3c213595e","Source":"/boson/fn","Type":"boson.fn"}
```

I know as programmers, it's nice to see if there are invisible characters at the end of the string or other small debugging concerns. It's just my opinion that in most cases, a cleaner and simpler view is better.

Reply:
I have removed the quotes around the keys, added a space after the colon, and removed metadat from the CloudEvents invocation.